src/components/EventHandler.js

```diff
@@ -55,7 +55,7 @@ export default class EventHandler extends React.Component {
     //
 
     handleScrollWheel(e) {
-        if (!this.props.enablePanZoom) {
+        if (!this.props.enablePanZoom && !this.props.enableDragZoom) {
```

Comment:
Hi @sartaj10. I'm curious about this line of code. Does drag zoom still utilize the scroll wheel? I noticed that the mouseDown and mouseUp events change behavior based on the selected prop type, but this one does not. This leads to zoom behavior. 

It might be the naming the is confusing me. `PanZoom` and `DragZoom` sound very similar, but it appears that they expand out at `Pan and Zoom` and `Drag to Zoom`. In that case, I wouldn't expect `Drag to Zoom` to have the zoom on scroll wheel.

Reply:
It does. You can see in the DDOS example - http://software.es.net/react-timeseries-charts/#/example/ddos

The text "pan and zoom" vs "drag to zoom" I think you're referring to the comments here - https://github.com/esnet/react-timeseries-charts/blob/41d1ceb1415565265c78f186ff5869818d46fa81/src/components/ChartContainer.js#L419 which could maybe be more clear.

The way I originally added this drag to zoom feature was a modifier key that was held down to perform drag-zoom. I only thought of it as changing the behavior of the mouse drag, which is why I didn't touch the mousewheel behavior.

I find still being able to use the mousewheel to zoom nice, but maybe an option to turn that off would be good? or adding a mod key back in to use whichever drag behavior you don't have turned on?

Reply:
Thanks @iabw . You are correct that the comments in 41d1ceb are where this idea of them being separate comes from.

I had expected that drag to zoom would enable that mouse up/down events, but would not affect scrolling. The reason that this matters in my case is that many charts on the screen and scrolling up / down becomes more difficult in that situation.

I'm not advocating for anything on behalf of the project, but did want to check if it was intentional and indicate that it was slightly confusing for me as a user of the library.

Reply:
Yes, I was just looking for this information for my implementation of the library. I'm trying to turn off the scroll wheel (`PanZoom`), while having drag to zoom enabled. I have a filter options on the side of the chart that allows the user of my application to chose which zoom they prefer. However, it seems that when the props for `enablePanZoom` is false and `enableDragZoom` is true, it will still operate with both the mouse scrolling and dragging to zoom. From this post, it seems this is expected, but just to clarify for the props' variables, is this expected behavior?